src/pages/instant-observability.js

```diff
@@ -65,11 +67,25 @@ const filterByContentType = (type) => (quickstart) => {
 
 /**
  * Filters a quickstart based on a category.
- * @param {String} type The category type (e.g. 'featured').
+ * @param {String} category The category type (e.g. 'featured').
  * @returns {(Object) => Boolean} Callback function to be used by filter.
  */
-const filterByCategory = (type) => (quickstart) => {
-  return !type || (quickstart.keywords && quickstart.keywords.includes(type));
+const filterByCategory = (category) => {
+  const associatedKeywords = CATEGORIES.find((cat) => cat.value === category)
+    ?.associatedKeywords;
+  const isValidCategory = category && associatedKeywords;
+
+  return (quickstart) => {
+    if (!isValidCategory) {
+      return true;
+    }
+
+    const catKeywords = new Set([...associatedKeywords]);
```

Comment:
Since this is comparing to arrays of keywords and all we care about is if they have an intersection. So using a set made the most sense to me, let me know if you have any other ides there!